packages/icons/manifest.json

```diff
@@ -90,6 +90,16 @@
       "category": "Action",
       "size": "24"
     },
+    {
+      "name": "download_flat",
+      "category": "Action",
+      "size": "24"
+    },
+    {
+      "name": "download_flat",
+      "category": "Action",
+      "size": "16"
+    },
```

Comment:
These should be named `download` and replace the current icon, like on [Figma](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC/SumUp-Iconography?node-id=8050%3A3777) (Internal link). We want to avoid moving away from the Figma source of truth and avoid duplicating icons (makes it unclear which one to use).

I think a minor is still fine since it's a cosmetic change